frontend/e2e/authenticated/special-uses/application-temp-outfitters.e2e-spec.ts

```diff
@@ -48,13 +48,21 @@ describe('Apply for a temp outfitters permit', () => {
   });
 
   it('should submit an application with only the required fields populated', () => {
+    element(by.css('.primary-permit-holder-first-name')).sendKeys('test');
```

Comment:
Updates to attempt to fix failing e2e tests. Not completely successful, though it appears these tests are NOT currently run in CI...